### ISSUE TITLE
Fusion Model

### ISSUE THREAD

**Issue by ethan92429** [AguaClara/horizontal_filtration#8]
@Galantino @ceo45 I just started a Fusion model to share ideas. I invited you, let me know if you didn't get it.

**Comment by ceo45** [comment]
Can you invite Corson too? cac468

**Comment by Galantino** [comment]
We've undermined you, Ethan. They will be called filter shelves from this point forward. Have a good day.

**Comment by ethan92429** [comment]
What are the filter shelves? The plate settlers? Is Monroe behind this for some technical reason I should know about?

**Comment by Galantino** [comment]
No, we just like filter shelves better. Minimizes confusion that can be had since they aren't necessarily plate settlers. Unless you disagree

**Comment by ethan92429** [comment]
wings, shelves, settlers, whatever floats you boat! (or sinks your sand, I should say 😺 )
